DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -251,56 +383,56 @@
 				<field name="" dataType="f32"/>
 			</Fields2>
 		</ActionID>
-		<ActionID ID="603" name="TextureAnimated()">
+		<ActionID ID="603" name="Animated Texture">
```

Comment:
Removing Function Style syntax in the name is a dangerous choice, however it doesn't serve much purpose to anyone who's not familiar to it, i suggest giving action ID's visual cues that allow to easily identify them.